service/pom.xml

```diff
@@ -24,6 +24,13 @@
       <artifactId>dropwizard-core</artifactId>
       <version>${dropwizard.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>net.sf.json-lib</groupId>
+      <artifactId>json-lib</artifactId>
+      <version>2.4</version>
+      <classifier>jdk15</classifier>
+    </dependency>
```

Comment:
This dependency is redundant. See: cc0b7135f7a2844943ec1b90e01f95fe37adb027

When you run `mvn clean package` are you trying to build just the service, or the entire project?